spec/board_spec.rb

```diff
@@ -0,0 +1,231 @@
+            subject.make_move(2, 0, "O")
+            subject.make_move(2, 1, "O")
+            subject.make_move(2, 2, "O")
+
+            expect(subject.find_winner).to eql('O')
+        end
+        it 'returns "X" when, given a board where X claims left column' do
+            subject.make_move(0, 0, "X")
+            subject.make_move(1, 0, "X")
+            subject.make_move(2, 0, "X")
+
+            expect(subject.find_winner).to eql('X')
+        end
+        it 'returns "X" when, given a board where X claims middle column' do
+            subject.make_move(0, 1, "X")
+            subject.make_move(1, 1, "X")
+            subject.make_move(2, 1, "X")
+
+            expect(subject.find_winner).to eql('X')
+        end
+        it 'returns "X" when, given a board where X claims right column' do
+            subject.make_move(0, 2, "X")
+            subject.make_move(1, 2, "X")
+            subject.make_move(2, 2, "X")
+
+            expect(subject.find_winner).to eql('X')
+        end
+        it 'returns "O" when, given a board where O claims left column' do
+            subject.make_move(0, 0, "O")
+            subject.make_move(1, 0, "O")
+            subject.make_move(2, 0, "O")
+
+            expect(subject.find_winner).to eql('O')
+        end
+        it 'returns "O" when, given a board where O claims middle column' do
+            subject.make_move(0, 1, "O")
+            subject.make_move(1, 1, "O")
+            subject.make_move(2, 1, "O")
+
+            expect(subject.find_winner).to eql('O')
+        end
+        it 'returns "O" when, given a board where O claims right column' do
+            subject.make_move(0, 2, "O")
+            subject.make_move(1, 2, "O")
+            subject.make_move(2, 2, "O")
+
+            expect(subject.find_winner).to eql('O')
+        end
+        it 'returns "X" when, given a board where X claims the right-left diagonal' do
+            subject.make_move(0, 0, "X")
+            subject.make_move(1, 1, "X")
+            subject.make_move(2, 2, "X")
+
+            expect(subject.find_winner).to eql('X')
+        end
+        it 'returns "X" when, given a board where X claims the left-right diagonal' do
+            subject.make_move(0, 2, "X")
+            subject.make_move(1, 1, "X")
+            subject.make_move(2, 0, "X")
+
+            expect(subject.find_winner).to eql('X')
+        end
+        it 'returns "O" when, given a board where O claims the right-left diagonal' do
+            subject.make_move(0, 0, "O")
+            subject.make_move(1, 1, "O")
+            subject.make_move(2, 2, "O")
+
+            expect(subject.find_winner).to eql('O')
+        end
+        it 'returns "O" when, given a board where O claims the left-right diagonal' do
+            subject.make_move(0, 2, "O")
+            subject.make_move(1, 1, "O")
+            subject.make_move(2, 0, "O")
+
+            expect(subject.find_winner).to eql('O')
+        end
```

Comment:
There might be a few more test cases here than necessary.  It's not a problem to have the extra ones, but you might find it helpful as the class gets bigger, if you reduce these tests to remove some duplication.

For example, it might be enough to test for one letter (X) on both diagonals (two test cases) and then have a 3rd test case to check the other letter (O).  You may not need to test both letters in both directions, to force the code to check all letters on all diagonals.